_plugins/jekyll-tool-tag.rb

```diff
@@ -18,7 +18,14 @@ def render(context)
       m = @text.match(format)
 
       if m
-        %Q(<span class="tool" data-tool="#{m[2]}" title="Tested with #{m[2]}"><strong>#{m[1]}</strong> <i class="fas fa-wrench" aria-hidden="true"></i><i aria-hidden="true" class="fas fa-cog"></i><span class="visually-hidden">Tool: #{m[2]}</span></span> )
+        # check if a variable was provided for the tool id
+        if context[m[2].tr('{}','')]
+          tool = context[m[2].tr('{}','')]
+        else
+          tool = m[2]
+        end
+
+        %Q(<span class="tool" data-tool="#{tool}" title="Tested with #{tool}"><strong>#{m[1]}</strong> <i class="fas fa-wrench" aria-hidden="true"></i><i aria-hidden="true" class="fas fa-cog"></i><span class="visually-hidden">Tool: #{tool}</span></span> )
```

Comment:
@hexylena this can probably be done nicer, I just wanted to have the tool tag take a variable so I can easily change the version of a tool in my tutorial after an update